pulsar-broker/src/main/java/org/apache/pulsar/broker/BookKeeperClientFactoryImpl.java

```diff
@@ -234,6 +232,13 @@ private void setEnsemblePlacementPolicy(ClientConfiguration bkConf, ServiceConfi
                     NET_TOPOLOGY_SCRIPT_FILE_NAME_KEY,
                     ""));
         }
+
+        if (conf.getBookkeeperClientIsolationGroups() != null && !conf.getBookkeeperClientIsolationGroups().isEmpty()) {
```

Comment:
We can use `StringUtils.isNotBlank` instead.

Reply:
changed the config check, Thx.